pytorch_lightning/strategies/parallel.py

```diff
@@ -125,6 +126,10 @@ def block_backward_sync(self):
         else:
             yield None
 
+    def launch(self, function: Callable, *args: Any, **kwargs: Any) -> Any:
+        launcher = SingleProcessLauncher()
+        return launcher.launch(function, *args, **kwargs)
```

Comment:
i don't think we should have this. this is unnecessary redirection for something that's supposed to execute inline.